waves/project.py

```diff
@@ -1592,19 +1610,25 @@ def energy_losses(
             by="turbine",
             units="kw",
         )
+
```

Comment:
This section should be roughly in the form of:

```python
if not losses_breakdown:
    production = ... # the code that's there already
    losses = potentail.copy()
    losses["total_losses"] = potential.values - production.values  # can use a different column name
else:
    # raise an error for if by == "turbine" because detailed breakdowns are only for the wind farm
    environmental_lr = self.environmental_loss_ratio()
    availability_lr = self.availability(which="energy", by="windfarm", frequency="month-year")
    ... # the remaining loss ratios

    base = potential.values
    environmental_losses = base - base * (1 - environmental_lr)
    availability_losses = base - el - base * (1 - environmental_lr) * (1 - availability_lr)
    ... # the remaining losses
    total_losses = ... # full calculation

    losses = potential.copy
    losses = losses.assign(
        environmental_losses=environmental_losses,
        availability_losses=availability_losses,
        ... # the remaining lossses
    ).rename(columns={}) # final step if you think we should have them displayed more nicely

# convert to desired units

# aggregate to apporpriate time scale ("annual", "project")

# apply aep

# apply per_capacity

Reply:
From our conversation: only calculate the loss ratio, not the actual losses. (removed from the example above)